Flex (#760)
shopware/docs: products/paas/repository.md

# Repository

The source code of your project will reside in a git-based VCS repository. You can start with a plain project, however we suggest starting with a new composer create-project. You will find out more about the setup template in the [Setup Template](setup-template.md) section.

{% hint style="info" %}
This guide explains the repository setup using **GitHub**. You can also integrate Bitbucket or GitLab based version control environments with Shopware PaaS. More information in [Source Integrations](https://docs.platform.sh/integrations/source.html).
{% endhint %}

## Create a Shopware project

First we create a new project with `composer create-project shopware/platform:dev-flex <folder-name>` using the [flex template](../../guides/installation/flex.md).

> This will create a brand new Shopware 6 project in the given folder. Now we have to change into our new created project and require the PaaS configuration with `composer req paas`.

Reviewer: ```suggestion
This will create a brand new Shopware 6 project in the given folder. Now, change it into the newly created project and require the PaaS configuration with `composer req paas`.
```

What do you mean by "require the PaaS configuration"? Is it written right?

> Second we have to create a new Git repository and push it to your favorite Git hosting service.

Reviewer: ```suggestion
Secondly, create a new Git repository and push it to your favorite Git hosting service.
```

## Add PaaS remote

> Next, we need to add a second remote, which allows us to push code towards the PaaS environment and trigger a deployment.

Reviewer: ```suggestion
Lastly, add a second remote, which allows us to push code towards the PaaS environment and trigger a deployment.
```
